src/components/inputs/SearchSelect.jsx

```diff
@@ -396,11 +396,12 @@ const SearchSelect = React.createClass({
       disabled,
     } = this.props;
     let { isFocused, isListOpen, filterValue } = this.state;
-    let selectedOptionId = this._getSelectedOptionId();
 
-    let filteredOptions = this._getFilteredOptions();
+    let selectedOptionId = this._getSelectedOptionId();
     let selectedOption = this._getOption(selectedOptionId);
 
+    let filteredOptions = isListOpen && this._getFilteredOptions();
```

Comment:
Since we only use `filterOptions` once, is it really useful to put it in a variable?